src/__test__/components/ReferralButton.test.jsx

```diff
@@ -6,9 +6,9 @@ import userEvent from '@testing-library/user-event';
 import fetchMock, { enableFetchMocks } from 'jest-fetch-mock';
 import '@aws-amplify/auth';
 
+import pushNotificationMessage from 'utils/pushNotificationMessage';
+import endUserMessages from 'utils/endUserMessages';
 import ReferralButton from '../../components/ReferralButton';
```

Comment:
Can you also fix the import of the `ReferralButton`?

Reply:
ok, for the record I just used global search and replace for `/redux` and `/utils` so I didn't go file by file, that's why it's not changed